README.md

```diff
@@ -26,6 +26,27 @@ const apos = require('apostrophe')({
 });
 ```
 
+#### Setting the `baseUrl`
+
+It is important to [set the `baseUrl` option](https://docs.apostrophecms.org/reference/core-server.html#baseurl) on your ApostropheCMS application for various reasons. In the SEO module it contributes to building the correct `canonical` link tag URL. This can be set on the main app configuration in `app.js` (staticly or with an environment variable) or [in the `data/local.js` file](https://docs.apostrophecms.org/core-concepts/global-settings/settings.html#changing-the-value-for-a-specific-server-only) as that file will contain environment/server-specific configurations.
```

Comment:
statically, not staticly. Otherwise great.

Reply:
doh